SS_GANGSYSTEM.pwn

```diff
@@ -1647,6 +1648,30 @@ CMD:ghelp(playerid)
 
 }
 
+/**
+* @method backup
+* @desc Calls all gang members for backup
+*
+* @param playerid {Number}
+*/
+CMD:backup(playerid)
+{
+    // Is a gang member
+    if (!IsPlayerInAnyGang(playerid)) return 1;
+
+    foreach (new p: SS_Player)
+    {
+        // Is gangemate
+        if (!strcmp(GInfo[playerid][gangname],GInfo[p][gangname])) continue;

```

Comment:
I think there's a problem in the condition, I gotta make sure it's right, if not will update the pull.